src/server_manager/web_app/ui_components/outline-server-settings.js

```diff
@@ -159,29 +77,17 @@ Polymer({
         display: block;
         margin-top: 6px;
       }
-      paper-checkbox {
-        /* We want the ink to be the color we're going to, not coming from */
-        --paper-checkbox-checked-color: var(--primary-green);
-        --paper-checkbox-checked-ink-color: var(--dark-gray);
-        --paper-checkbox-unchecked-color: var(--light-gray);
-        --paper-checkbox-unchecked-ink-color: var(--primary-green);
-      }
-      .selection-container paper-checkbox {
-        margin-right: 4px;
-      }
     </style>
     <div class="container">
       <div class="content">
-        <!-- DO information -->
-        <div class="setting card-section" hidden\$="[[!isServerManaged]]">
-          <img class="setting-icon digital-ocean-icon" src="images/do_white_logo.svg">
-          <div>
-            <h3>DigitalOcean</h3>
-            <paper-input readonly="" value="[[_getShortName(cloudLocation, localize)]]" label="[[localize('settings-server-location')]]" hidden\$="[[!cloudLocation]]" always-float-label="" maxlength="100"></paper-input>
-            <paper-input readonly="" value="[[serverMonthlyCost]]" label="[[localize('settings-server-cost')]]" hidden\$="[[!serverMonthlyCost]]" always-float-label="" maxlength="100"></paper-input>
-            <paper-input readonly="" value="[[serverMonthlyTransferLimit]]" label="[[localize('settings-transfer-limit')]]" hidden\$="[[!serverMonthlyTransferLimit]]" always-float-label="" maxlength="100"></paper-input>
-          </div>
-        </div>
+        <outline-do-info-setting hidden\$="[[!_isDo(cloudId)]]"
```

Comment:
As mentioned in the other file, there's no need for the separate elements. You can pass the cloud name and icon.

Reply:
Done